ImportBlocks/ImportBlocks.cs

```diff
@@ -4,52 +4,32 @@
 using Neo.Network.P2P.Payloads;
 using Neo.Persistence;
 using System;
-using System.Collections.Generic;
 using System.IO;
-using System.IO.Compression;
-using System.Linq;
-using System.Text.RegularExpressions;
 
 namespace Neo.Plugins
 {
     public class ImportBlocks : Plugin
     {
+        private IActorRef _blockImporter;
         public ImportBlocks()
         {
-            OnImport();
+            _blockImporter = System.ActorSystem.ActorOf(BlockImporter.Props());
         }
 
-        private static bool CheckMaxOnImportHeight(uint currentImportBlockHeight)
+        public override void Configure()
         {
-            if (Settings.Default.MaxOnImportHeight == 0 || Settings.Default.MaxOnImportHeight >= currentImportBlockHeight)
-                return true;
-            return false;
+            Settings.Load(GetConfiguration());
         }
 
-        public override void Configure()
+        protected override void OnPluginsLoaded()
         {
-            Settings.Load(GetConfiguration());
+            SuspendNodeStartup();
+            _blockImporter.Tell(new BlockImporter.StartImport { BlockchainActorRef = System.Blockchain, OnComplete = OnImportComplete});
```

Comment:
@jsolman, why there is ImportCompleted and ImportComplete?

Reply:
`Blockchain.ImportCompleted` is what is sent back when a `Blockchain.Import` message finishes processing. The `BlockImporter` will issue many `Blockchain.Import` messages and get back many `Blockchain.ImportCompleted` messages. When all importing is complete it will call back whatever action is passed in the `OnComplete` member of the `BlockImporter.StartImport` message. In this case it is passing the `OnImportComplete` method to be called when all importing is finished.